docs/api.md

```diff
@@ -65,18 +65,18 @@ const FingerprintJS = require('@fingerprintjs/fingerprintjs')
 
 ## API
 
-#### `FingerprintJS.load({ delayFallback?: number }): Promise<Agent>`
+#### `FingerprintJS.load({ delayFallback?: number, debug?: boolean }): Promise<Agent>`
 
 Builds an instance of Agent and waits a delay required for a proper operation.
 We recommend calling it as soon as possible.
 `delayFallback` is an optional parameter that sets duration (milliseconds) of the fallback for browsers that don't support [requestIdleCallback](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback);
 it has a good default value which we don't recommend to change.
+`debug: true` prints debug messages to the console.
 
-#### `agent.get({ debug?: boolean }): Promise<object>`
+#### `agent.get(): Promise<object>`
```

Comment:
Are we going to release a new major version? From my point of view, [according to the version policy](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/version_policy.md), this is a breaking change, isn't it? Or do I miss anything?

Reply:
It's not a breaking change because the parameter isn't actually removed. The parameter is [marked as deprecated](https://github.com/fingerprintjs/fingerprintjs/pull/666/files#diff-49cc4f0909411e6de89a20d5a97f81c481546b48973288ba70aeb002b0dd4138R32). The old code like this will work, but in a slightly other way:

```js
const fp = await fpPromise
const result = await fp.get({ debug: true })
console.log(result.visitorId)
```

So nothing will break after the update, therefore it's ok to keep the major version.

Reply:
I see it now, looks good, thank you for the clarification.